### PR TITLE
Add Fastlane Plugin option to RN Hermes sourcemaps page

### PR DESCRIPTION
**Pull request by krystofwoldrich**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

Related to https://github.com/getsentry/sentry-fastlane-plugin/pull/195

Adding the Fastlane example to show that Hermes source map and bundle have to be uploaded in one action together.

That is a difference compared to JSC.

### CODE REVIEW

File: src/platforms/react-native/manual-setup/hermes.mdx [hunk 1]
```diff
@@ -129,3 +129,21 @@ You will upload the `index.android.bundle` and `index.android.bundle.map` for An
...
+```ruby
+sentry_upload_sourcemap(
+  auth_token: 'YOUR_AUTH_TOKEN',
+  org_slug: '___ORG_SLUG___',
+  project_slug: '___PROJECT_SLUG___',
+  version: '...',
+  app_identifier: '...', # pass in the bundle_identifer of your app
+  build: '...', # Optionally pass in the build number of your app
+  dist: '...', # optional distribution of the release usually the buildnumber
+  sourcemap: ['main.jsbundle', 'main.jsbundle.map'], # Sourcemap(s) to upload. Path(s) can be a comma-separated string or an array of strings.
```
Comment (marandaneto) [inline src/platforms/react-native/manual-setup/hermes.mdx:146]: What's about the Android ones? shouldn't they be added as well?

File: src/platforms/react-native/manual-setup/hermes.mdx [hunk 2]
```diff
@@ -129,3 +129,35 @@ You will upload the `index.android.bundle` and `index.android.bundle.map` for An
 When uploaded, the `index.android.bundle` file will be sized at 0 bytes. This is expected with Hermes bytecode and won't affect the source maps resolution.
 
 </Note>
+
+### Upload using Sentry Fastlane Plugin
```
Comment (lizokm) [inline src/platforms/react-native/manual-setup/hermes.mdx:133]:
```suggestion
### Upload Using Sentry Fastlane Plugin
```

File: src/platforms/react-native/manual-setup/hermes.mdx [hunk 3]
```diff
@@ -129,3 +129,35 @@ You will upload the `index.android.bundle` and `index.android.bundle.map` for An
...
+
+To upload generated sourcemaps using Fashlane you can use `sentry_upload_sourcemap` action. See how to install Sentry Fastlane Plguin [here](https://github.com/getsentry/sentry-fastlane-plugin).
```
Comment (lizokm) [inline src/platforms/react-native/manual-setup/hermes.mdx:135]:
```suggestion
To upload generated sourcemaps using Fastlane, use the `sentry_upload_sourcemap` action. See how to install Sentry Fastlane Plugin [here](https://github.com/getsentry/sentry-fastlane-plugin).
```

**Review by lizokm — APPROVED** [review]
A few small tweaks.

**Review by marandaneto — APPROVED** [review]

**Pull request merged**